src/network/udp.py

```diff
@@ -145,8 +145,9 @@ def handle_write(self):
                 self.write_buf, ('<broadcast>', self.port))
         except socket.error as e:
             logger.error("socket error on sendto: %s", e)
-            if e.errno == 101:
-                self.announcing = False
-                self.socket.close()
             retval = 0
```

Comment:
remove line 148

Reply:
Still not done (line 148 in new, 151 in old)

Reply:
removed line 148